subsys/net/lib/sockets/sockets.c

```diff
@@ -680,7 +690,7 @@ int _impl_zsock_fcntl(int sock, int cmd, int flags)
 	switch (cmd) {
 	case F_GETFL:
 		if (sock_is_nonblock(ctx)) {
-		    return O_NONBLOCK;
```

Comment:
don't fix that in this patch, it's not related at all. (you are not supposed to fix existing indentation issues). Feel free to send one separate if you want though.

Reply:
done